Doc/library/tkinter.rst

```diff
@@ -447,7 +447,7 @@ is helpful.
 Navigating the Tcl/Tk Reference Manual
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-As noted, the official `Tk commands <https://www.tcl.tk/man/tcl8.6/TkCmd/contents.htm>`_
+As noted, the official `Tk commands`_
```

Comment:
This addition fixes: `Warning: /home/runner/work/cpython/cpython/Doc/library/tkinter.rst:2: WARNING: Duplicate explicit target name: "tk commands".` // `make: *** [Makefile:49: build] Error 1`